index.html

```diff
@@ -1614,24 +1658,19 @@ <h5>Reading Progression Direction</h5>
 							<li><code><dfn>rtl</dfn></code>: right-to-left.</li>
 						</ul>
 
-						<p>The default value is <code>ltr</code>.</p>
+						<p>The default value is <code>ltr</code>. If the <code>readingProgression</code> is not set,
+							user agents MUST use the default value when generating their <a>canonical
+							representation</a>.</p>
 
 						<p>This property has <em>no effect</em> on the rendering of the individual primary resources; it
 							is only relevant for the progression direction from one resource to the other.</p>
 
-						<p class="note">The reading progression of a publication is used to adapt such publication level
-							interactions as menu position, swap direction, defining tap zones to lead the user to the
-							next and previous pages, touch gestures, etc.</p>
-
-						<p>If the <code>readingProgression</code> is not set, user agents MUST use the default value
-								<code>ltr</code> when generating their <a>canonical representation</a>.</p>
-
 						<pre class="example" title="Reading progression set explicitl to ltr">
 {
     "@context"           : ["https://schema.org","https://www.w3.org/ns/pub-context"],
     "type"               : "Book",
     &#8230;
-    "url"                : "https://publisher.example.org/mobydick",
+    "url"                : "https://publisher.example.org/leviathan",
     "readingProgression" : "ltr"
 }
 </pre>
```

Comment:
Yes, let's take this up in #51, though. I'm fine adding another value now if it's reasonable that a top-to-bottom progression will be needed, otherwise we need to clarify extensibility as right now the wording wouldn't allow for other values.